test/functional/feature_rbf.py

```diff
@@ -624,6 +627,24 @@ def test_no_inherited_signaling(self):
         assert_equal(True, self.nodes[0].getmempoolentry(optin_parent_tx['txid'])['bip125-replaceable'])
         assert_raises_rpc_error(-26, 'txn-mempool-conflict', self.nodes[0].sendrawtransaction, replacement_child_tx["hex"], 0)
 
+    def test_replacement_relay_fee(self):
+        outpoint = make_utxo(self.nodes[0], int(1.1*COIN))
+
+        initial_transaction = CTransaction()
+        initial_transaction.vin = [CTxIn(outpoint, nSequence=0)]
+        initial_transaction.vout = [CTxOut(1 * COIN, DUMMY_P2WPKH_SCRIPT)]
+        initial_transaction_hex = txToHex(initial_transaction)
+        self.nodes[0].sendrawtransaction(initial_transaction_hex, 0)
+
+        # Higher fee, higher fee per KB, but the replacement does not provide a relay
+        # fee conforming to node's `incrementalrelayfee` policy of 1000 sat per KB.
+        replacement_transaction = CTransaction()
+        replacement_transaction.vin = [CTxIn(outpoint, nSequence=0)]
+        replacement_transaction.vout = [CTxOut(int(1 * COIN - 1), DUMMY_P2WPKH_SCRIPT)]
```

Comment:
how is this test different from the one above? `# Should fail because we haven't changed the fee`

Reply:
It is different because the newly added test adds coverage for previously uncovered code https://marcofalke.github.io/btc_cov/total.coverage/src/validation.cpp.gcov.html